// Copyright 2025 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

edition = "2024";

option optimize_for = LITE_RUNTIME;

package proto_extras;

message TestMessageEdition {
  string text = 1;
}
